### PR TITLE
Added a new endpoint to retrieve the container entity id from the pod uid and the container name

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Added a new endpoint to retrieve the container entity id from the pod uid and the container name

### Motivation

Be able to collect container tags for logs.

### CODE REVIEW

File: pkg/util/kubernetes/kubelet/kubelet.go [hunk 1]
```diff
@@ -250,6 +250,29 @@ func (ku *KubeUtil) GetPodForEntityID(entityID string) (*Pod, error) {
 	return ku.GetPodForContainerID(entityID)
 }
 
+// GetContainerEntityID returns the container entity id matching the podUID and containerName,
+// returns an error if the arguments are invalid or if no container is found.
+func (ku *KubeUtil) GetContainerEntityID(podUID string, containerName string) (string, error) {
+	pods, err := ku.GetLocalPodList()
```
Comment (xvello) [inline pkg/util/kubernetes/kubelet/kubelet.go]: we could reuse `GetPodFromUID` here, in order to use the caching layer

File: pkg/util/kubernetes/kubelet/kubelet.go [hunk 2]
```diff
@@ -250,6 +250,29 @@ func (ku *KubeUtil) GetPodForEntityID(entityID string) (*Pod, error) {
...
+	if err != nil {
+		return "", err
+	}
+	if podUID == "" || containerName == "" {
```
Comment (xvello) [inline pkg/util/kubernetes/kubelet/kubelet.go]: Might as well do it at the very top of the method?

**Review by xvello — COMMENTED** [review]
Good addition, two nits

**Review by xvello — APPROVED** [review]
LGTM 👍

**Review by moisesbotarro — APPROVED** [review]
lgtm

**Pull request closed without merging**